modules/virt-preparing-container-disk-for-vms.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+[id="virt-preparing-container-disk-for-vms_{context}"]
+= Preparing a container disk for virtual machines
+
+A container disk must be built with a virtual machine image and pushed to a container registry. It can then either be imported into a DataVolume and attached to a virtual machine or attached directly to a virtual machine as an ephemeral containerDisk volume.
+
+.Prerequisites
+
+* You have root or sudo privileges on your local system to install podman.
+
+* The virtual machine image must be either QCOW2 or RAW format.
+
+.Procedure
+
+. Install `podman` if it is not already installed:
++
+[source,terminal]
+----
+$ sudo yum install podman -y
+----
+
+. Create a Dockerfile to build the virtual machine image into a container image.
+The virtual machine image must be owned by QEMU, which has a UID of `107`, and placed in the `/disk/` directory inside the container. Permissions for the `/disk/` directory must then be set to `0440`.
+
+The following example uses the Red Hat Universal Base Image (UBI) to handle these configuration changes in the first stage, and uses the minimal `scratch` image in the second stage to store the result.
++
+[source,terminal]
+----
+$ cat > Dockerfile << EOF
+FROM registry.access.redhat.com/ubi8/ubi:latest AS builder
+ADD --chown=107:107 <vm_image>.qcow2 /disk/ <1>
+RUN chmod 0440 /disk/*
+
+FROM scratch
+COPY --from=builder /disk/* /disk/
+EOF
+----
+<1> Where <vm_image> is the virtual machine image in either QCOW2 or RAW format.+
+To use a remote virtual machine image, replace `<vm_image>.qcow2` with the complete url for the remote image.
+
+. Build and tag the container:
```

Comment:
Possibly due to the above `+`, these last two steps are erroneously being nested underneath the callout description.

Reply:
Ah yep, missing + before the UBI para